server/manager/src/main/java/org/apache/accumulo/manager/tserverOps/ShutdownTServer.java

```diff
@@ -57,6 +57,13 @@ public long isReady(long tid, Manager manager) {
       return 0;
     }
 
+    // if one tserver is remaining throw an error and return before attempting shutdown
+    if (manager.onlineTabletServers().size() == 1) {
+      log.error("Only 1 tserver running and it contains the Root table. Not shutting down {}. ",
+          server);
+      return 0;
```

Comment:
I'm not sure that this is doing what the comment suggests. Returning 0 means that the FateOp is ready for execution and the `call` method will be executed.

Reply:
Nevermind, I see what's happening here. Might want to adjust the comment to say that by returning early the `manager.shutdownTServer` call is never made and the tserver will not be shut down.

Reply:
Yeah I think we want to throw an error instead, to break out of the operation.

Reply:
So it works like this but the client thinks that it succeeded. Not sure what happens to the FATE TX. I am going to test it with an error being thrown.